src/App.vue

```diff
@@ -10,6 +11,7 @@ import SiteHeader from './common/components/SiteHeader.vue'
       <nav>
         <RouterLink to="/">Home</RouterLink>
       </nav>
+      <DropDown/>
```

Comment:
Cool dropdown bro, but why?

Reply:
tbh, just started to create site objects and views at random, in hopes they can be picked up and used as a starting point in further development - my idea was to start by making the flow and structure that isn't polished, then add functionality and the polish, but maybe implementing functionality or something i haven't considered might make the process a painful...